copier.yml

```diff
@@ -62,8 +62,29 @@ component_owner:
     type: str
     help: |
         Owner of the component in the Developer Portal Software Catalogue.
-        This will normally be group:default/<your_group>
-    default: "group:default/sscc"
+        This will normally be group:default/<your group>
```

Comment:
It would be good to include a link in here to a list of available options. [This](https://dev-portal.diamond.ac.uk/catalog?filters%5Bkind%5D=group&filters%5Buser%5D=all) comes close but the button(s) to get the actual possible values for this string are not obvious.

Reply:
Can't think of anything better tbh. At least it indicates it's not just a string. Perhaps worth phrasing it as `The unique name of the person or group who own this component`

Reply:
Seems to be `group:default/xxx` where `xxx` is last part of the URL if you hover over the options in your list...

Reply:
Indeed, there is enough information to work it out, I was hoping for a very clear list of options though. Again I think that can be left for a future PR if we ever get one.

Reply:
I think putting a link to https://dev-portal.diamond.ac.uk/catalog?filters%5Bkind%5D=group&filters%5Buser%5D=all in the question and telling people to hover over the links in it to get the list of valid groups would be useful

Reply:
Done and also made an issue to simplify: https://github.com/DiamondLightSource/python-copier-template/issues/191